README.md

```diff
@@ -349,6 +349,19 @@ bundle exec cap development deploy
 
 #### Deploying They Vote For You to production
 
+We now have two productions servers, and a blue/green deployment process driven
```

Comment:
Oops. Looks like this got added to the They Vote For You section rather than the Planning Alerts section